control/lacp.yaml

```diff
@@ -1,32 +1,43 @@
 openapi: 3.1.0
 
 info:
-  title: LACP Control models
+  title: LACP State models
   description: >-
-    All LACP control schemas
+    All LACP State schemas
   version: ^0.0.0
 
 components:
   schemas:
-    LacpMember.State:
+    Lacp.State:
       description: >-
-        Set LACP state for specified LAG Member Port(s).
+        Set states specific to LACP
+      type: object
+      properties:
+        choice:
+          type: string
+          enum: 
+          - admin_state
+        admin_state:
+          $ref: '#/components/schemas/Lacp.Admin.State'
+    Lacp.Admin.State:
+      description: >-
+        Set LACP admin state for specified LAG Member Port(s).
       type: object
       required: [state]
       properties:
-        lag_member_port_names:
+        port_names:
           description: >-
             The names of LAG members (ports) for which the state has to be applied.
-            An empty or null list will control all LAG members.
+            An empty or null list will control all ports that are part of a LAG.
           type: array
           items:
             type: string
           x-constraint:
           - '/components/schemas/Port/properties/name'
         state:
           description: >-
-            The LACP Member admin state.
+            The LACP member admin state.
             'up' will start transmission of LACPDUs on selected member ports.
```

Comment:
I reviewed the descriptions with Subrata/Sourav since I was under the impression that this is no longer accurate.
Can you please change the 'up' and 'down' description to below ( as received from Subrata ) which is more accurate than what was added initially:
'up' will turn on ‘sync’ flag of LACPDUs transmitted from selected member ports.
'down' will turn off ‘sync’ flag of LACPDUs transmitted from selected member ports.